src/fu-binder-aidl.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2024 Collabora Ltd. <colin.kinloch@collabora.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#include <gbinder.h>
+
+enum fu_binder_calls {
```

Comment:
any reason these can't be in alphabetical order?

Reply:
The order was taken from the DBus interface xml.
However things have moved along and all that matters it that the order is the same between `IFwupd.aidl` and `method_funcs` in `fu-binder-daemon.c` and that's still work in progress:
https://github.com/fwupd/fwupd/blob/742ae8f10a367bf14e6ac76b438a8eb1fc2043fd/contrib/android/aidl/org/freedesktop/fwupd/IFwupd.aidl#L5-L9
https://github.com/fwupd/fwupd/blob/742ae8f10a367bf14e6ac76b438a8eb1fc2043fd/src/fu-binder-daemon.c#L349-L354